### DISCUSSION TITLE
Dropdown issue in production enviroment with Next Js (V 3.5.1)

### DISCUSSION THREAD

**Discussion by hussein-haitham** [saadeghi/daisyui#2272] [Q&A]
I encountered a problem with the dropdown menu items in production, where padding and hover effect on menu item not working. however in dev mode its working fine just like presented in the main website. 

 I switched my project from version 2.5 to 3.5 and this bug only occured in production. And i cannot find asolution for it so far.
 I even tested the dropdown compnent on stackblitz seperatley and also the problem persists in NextJs production

**Comment by technophile-04** [comment]
I think https://github.com/saadeghi/daisyui/discussions/2183 here also they got similar issue, but don't how it was solved

**Reply by hussein-haitham** [reply]
So far I couldn't solve the issue itself but I made a work around on providing a styling class to the dropdown items and provide custom properties.

**Comment by saadeghi** [comment]
Are you using the latest version of Next.js? If not, please update to the latest version of Next.js and check if the issue still exists?

**Reply by hussein-haitham** [reply]
I'm using Next 13.1 with pages directory

**Reply by saadeghi** [reply]
the latest version is `13.4`
